.github/workflows/ci.yml

```
@@ -433,9 +511,37 @@
     needs: manual
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - name: redis-latest
-      run: docker build . --file ${{ env.DOCKERFILE_REDIS }} --tag "redis-latest:$(date +%s)"
+    - name: Check out the repo
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)